internal/deviations/deviations.go

```diff
@@ -625,3 +625,9 @@ func ISISLspMetadataLeafsUnsupported(dut *ondatra.DUTDevice) bool {
 func QOSQueueRequiresID(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosQueueRequiresId()
 }
+
+// EcnSameMinMaxThresholdUnsupported returns true for devices that don't support Minimum and Maximum threshold values are same
+// CISCO: minimum and maximum threshold value not be same. Anything less than maximum threshold should be 6144 greater than minimum threshold value.
```

Comment:
... values cannot be the same; anything ...

Reply:
modified